bin/Cores/cores.json

```diff
@@ -361,15 +361,26 @@
   },
   "mednafen_pce_libretro":{
     "name": "Beetle PCE",
-    "extensions": "pce|cue",
+    "extensions": "pce",
```

Comment:
While the oversight wasn't added by you, I recommend adding sgx to Beetle PCE while you're in the neigborhood, since only PCE FAST fails to support supergrafx.

Reply:
Added "sgx" to Beetle PCE. Also added "j64|rom" to Virtual Jaguar since by default it was only looking for j64.